### ISSUE TITLE
Incorrectly rewrites URL scheme

### ISSUE THREAD

**Issue by Nateowami** [bobby-brennan/rss-parser#14]
I used the example code locally (serving from the file system) which fetches `https://www.reddit.com/.rss`, and I'm getting

    rss-parser.js:5812 Fetch API cannot load file://www.reddit.com/.rss. URL scheme must be "http" or "https" for CORS request.

That's using `dist/rss-parser.js` in version 2.5.0.

---
If I downgrade to 2.4.0, (which in the `/dist/rss-parser.js` file is labeled 2.2.4), I get the following error:

    Fetch API cannot load https://www.reddit.com/.rss. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'null' is therefore not allowed access. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

(Initially I didn't realize you updated it and wondered why I was suddenly getting a different error).

**Comment by Nateowami** [comment]
Just a note that the problem in 2.4.0 is most likely due to the server not allowing CORS, period. So probably not actually a problem in this library.

**Comment by andjosh** [comment]
Strange. According to the docs, the `protocol` has a default set already. I could change it to explicitly pass the parsed value, though.

~~~js
var parsedUrl = url.parse(feedUrl);
var req = get({
    protocol: parsedUrl.protocol, // add this
    hostname: parsedUrl.hostname,
    path: parsedUrl.path,
    headers: {'User-Agent': 'rss-parser'}
  }, function(res) {
});
~~~

**Comment by bobby-brennan** [comment]
browserify is dark magic.

@Nateowami can you try `bower install rss-parser#protocol-fix` and see if it fixes the problem?  That incorporates the fix from andjosh above

**Comment by Nateowami** [comment]
@bobby-brennan Thanks. The example code works with `rss-parser#protocol-fix`.

**Comment by bobby-brennan** [comment]
Great, fix released as 2.5.1
